lib/install.mjs

```diff
@@ -0,0 +1,132 @@
+import { createWriteStream, existsSync, rmSync, renameSync } from 'node:fs';
```

Comment:
@TheJaredWilcurt Does it matter if I use sync methods?

Reply:
With an exists check or a rename, this is generally a very fast process and I think you can "get away with it", in most cases. However, a recursive folder delete can be noticeably slow. Since this script gets ran after an `npm install` there may be other packages also doing post-installs at the same time. The usage of sync at all could be blocking for other package's post-install scripts (Node is single threaded).

Because of this, I think going fully async with all of these would be the best practice for end users and could help speed up the total install time depending on the user's project.

Reply:
Makes sense to me, thanks for the explanation!